introspection-engine/connectors/mongodb-introspection-connector/src/sampler.rs

```diff
@@ -50,6 +53,12 @@ pub(super) async fn sample(
 
     let data_model = statistics.into_datamodel(&mut warnings);
 
+    let data_model = if render_config {
+        psl::render_datamodel_and_config_to_string(&data_model, config)
+    } else {
+        psl::render_datamodel_to_string(&data_model, Some(config))
+    };
```

Comment:
Should we leave a comment somewhere saying that the render_config should always be true, and we should refactor everywhere where it's false to be true?

Reply:
done